.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 sudo: false
 node_js:
-  - "6.10.3"
+  - "8.1.0"
```

Comment:
Nope, it should work with at least node v4 and up – but in our CI builds node is mostly an auxiliary for running the builds, scripts & installing packages etc. Doesn't make much sense to also run the CI with v6 & v8 to me, especially as that would also amplify our build times (which are already at 60+min) by a few times that.